ws-nextjs-app/pages/[service]/send/[id]/send.stories.tsx

```diff
@@ -31,4 +38,5 @@ export default {
   Component,
 };
 
-export const Example = Component;
+export const Form = () => <Component initialScreen="form" />;
+export const Uploading = () => <Component initialScreen="uploading" />;
```

Comment:
Is it overkill to have stories for the success & error states?

Reply:
We have separate tasks for the success and error states, so was just going to leave any stories for those individual tickets.